cpp/autosar/src/rules/A8-4-7/InParametersForCheapToCopyTypesNotPassedByValue.ql

```diff
@@ -34,7 +35,8 @@ where
   ) and
   t.isConst() and
   not exists(CatchBlock cb | cb.getParameter() = v) and
-  not exists(CopyConstructor cc | cc.getAParameter() = v) and
+  not exists(SpecialMemberFunction cc | cc.getAParameter() = v) and
+  not exists(Operator op | op.getAParameter() = v) and
```

Comment:
The alert message is not appropriately formatted. Can you replace `" + v.getName() + "` with `'" + v.getName() + "'` and`" + t.getName() + "` with `'" + t.getName() + "'`